docker-compose.yaml

```diff
@@ -76,7 +76,7 @@ services:
 
   mapi:
     container_name: mapi
-    image: bitcoinsv/mapi:1.3.0
+    image: 709855634846.dkr.ecr.eu-west-2.amazonaws.com/mapi:1.4.1
```

Comment:
this repo suppose to be public while the image mentioned here is private. We wouldn't let a public repo to checkout private info